File: B/Python/Product.py
class Product():
    __id = ""
    __price = 0
    
    def __init__(self):
        self.__id = ""
        self.__price = 0
        
    def setID(self, i):
        self.__id = i
    
    def getID(self):
        return self.__id
    
    def setPrice(self, p):
        self.__price = p
    
    def getPrice(self):
        return self.__price

File: A/Product.java
public class Product{
    private String id;
    private int price;

    public Product(){
        this.id = "";
        this.price = 0;
    }

    public void setID(String i){
        this.id = i;
    }

    public String getID(){
        return id;
    }

    public void setPrice(int p){
        this.price = p;
    }

    public int getPrice(){
        return price;
    }
}